app/services/action_queue.rb

```diff
@@ -74,12 +74,27 @@ def payload
           },
           action: {
             source: data[:source]
-          },
+          }.tap do |action|
+            if @action.form_data['is_subscription']
+              action[:skip_confirmation] = 1 if @action.form_data['recurrence_number'].to_i > 0
+              action[:fields] = action_fields
+            end
+          end,
+
           user: user_data
         }
       }
     end
 
+    def action_fields
+      {
+        recurring_id:      @action.member_id,
```

Comment:
This is for the data team. They need `recurring_id` to have a unique value per member.